.travis.yml

```diff
@@ -90,15 +90,15 @@ jobs:
         - . bin/include/versioning
         - ./bin/build-helm
         - mkdir helm-charts
-        - cp helm/quarks-job*.tgz helm-charts/
+        - cp -a helm/quarks-job*.tgz helm-charts/
```

Comment:
Isn't `cp --preserve=timestamps` or `cp -p` to preserve mode/timestamp? But I guess `-a` does as well